sdk/cosmosdb/cosmos/src/queryExecutionContext/EndpointComponent/NonStreamingOrderByEndpointComponent.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { DiagnosticNodeInternal } from "../../diagnostics/DiagnosticNodeInternal";
+import { Response } from "../../request";
+import { ExecutionContext } from "../ExecutionContext";
+import { OrderByComparator } from "../orderByComparator";
+import { NonStreamingOrderByResult } from "../nonStreamingOrderByResult";
+import { NonStreamingOrderByPriorityQueue } from "../../utils/nonStreamingOrderByPriorityQueue";
+import { getInitialHeader } from "../headerUtils";
+export class NonStreamingOrderByEndpointComponent implements ExecutionContext {
+  private nonStreamingOrderByPQ: NonStreamingOrderByPriorityQueue<NonStreamingOrderByResult>;
+  private isCompleted: boolean = false;
+  /**
+   * Represents an endpoint in handling an non-streaming order by query. For each processed orderby
+   * result it returns 'payload' item of the result
+   *
+   * @param executionContext - Underlying Execution Context
+   * @hidden
+   */
+  constructor(
+    private executionContext: ExecutionContext,
+    private sortOrders: any[],
+    private priorityQueueBufferSize: number = 2000,
```

Comment:
Let's move this constant value to a constant file.

Reply:
removed from here. Was already added as a constant.